src/lib/index.ts

```diff
@@ -3,11 +3,12 @@ import * as DeployUtil from './DeployUtil';
 import * as Keys from './Keys';
 import * as Serialization from './Serialization';
 import * as Signer from './Signer';
+import { Result } from 'ts-results';
```

Comment:
NIT: How about moving this line to the top of the file since it is an external library import :)

Reply:
moved

Reply:
LGTM 👍